CONTRIBUTING.md

```diff
@@ -30,6 +30,24 @@ Same rules apply here as for bug reports and feature requests. Plus:
 * We also prefer adding [emoji prefixes to commit messages](https://gitmoji.carloscuesta.me/). Since
   the the `gitmoji` CLI tool can be very [slow](https://github.com/zeenix/gimoji#rationale), we
   recommend using [`gimoji`](https://github.com/zeenix/gimoji) instead.
+* Add a prefix indicating the packages being changed.
+  E.g. for a commit changing the packages `zbus` and `zvariant`, prefix the commit message with `zb,zv: `.
```

Comment:
The prefix could be the package name too, the abbreviations are just for convenience.